src/components/CategoryPage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CategoryPage = () => (
+  <div>
+    <h1>Coming Soon!!</h1>
```

Comment:
- [ ] Let's follow the guidelines by displaying only a button with text "Check status" here.

Reply:
@ignatius22 
Done

Reply:
@ignatius22 
Done